### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by alexec**
Issue #1095

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
-## Requirements
-Make sure you have following tools installed
+## Before You Start
+
+You must install and run the ArgoCD eusing miniubke first. This will help you understand the application, but also get your local environment set-up.
```
Comment (alexmt) [inline CONTRIBUTING.md]: `eusing` -> `using`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -33,45 +52,55 @@ $ go get -u github.com/argoproj/argo-cd
...
-## Running locally
+## Running Locally
 
-You need to have access to kubernetes cluster (including [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) or [docker edge](https://docs.docker.com/docker-for-mac/install/) ) in order to run Argo CD on your laptop:
+It is much easier to run and debug if you run ArgoCD on your local machine than in the Kubernetes cluster.
 
-* install kubectl: `brew install kubectl`
-* make sure `kubectl` is connected to your cluster (e.g. `kubectl get pods` should work).
-* install application CRD using following command:
+You should scale the deployemnts to zero via the Kubernetes dashboard, then start the services:
```
Comment (alexmt) [inline CONTRIBUTING.md]: I think most people use `kubectl` instead of dashboard.

**Review by alexmt — APPROVED** [review]
LGTM. Please take a look at couple minor comments before merging.

**Comment by alexec** [comment]
I'll keep this open for a bit as I add more details.

**Review by jessesuen — APPROVED** [review]

**Review by alexmt — APPROVED** [review]

**Comment by alexec** [comment]
@jessesuen / @alexmt - could you please merge this PR? I do not have permission.

**Pull request merged**